test/basic-querying.test.js

```diff
@@ -593,6 +613,19 @@ describe('basic-querying', function() {
 
     describe('geo queries', function() {
       describe('near filter', function() {
+        it('supports a near query with null condition', function(done) {
```

Comment:
if i'm right in my previous comment the test name should also be revised to reflect that we are not querying on a null condition, but that results on the conditions could come with empty properties

something like:
`'supports a compound near query with conditions, allowing results where conditions match null properties'`